web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -712,6 +716,72 @@ export function DatabaseItem(props: SearchResultItem<SearchResultDatabase>) {
   );
 }
 
+export function AppItem(props: SearchResultItem<SearchResultApp>) {
```

Comment:
Let's add a couple of examples of it to the story, just like we do with other resources. I'm particularly interested in a version with long values – keep in mind that for dbs we have five different variants with five different lengths.